scripts/config-list.js

```diff
@@ -41,6 +41,7 @@ var output = {
   limits: {},
   allowedIPs: [],
   allowedEmailDomains: [],
+  allowedPhoneNumbers: [],
```

Comment:
IIUC, I think we also need to add logic to actually fetch `allowedPhoneNumbers` from memcache, similar to `mc.getAsync('allowedEmailDomains')`?

Reply:
[Added](https://github.com/mozilla/fxa-customs-server/pull/218/files#diff-51918078d313a138e042691cb119ed4dR79) - mind having a look?